cargo-fix/src/cli.rs

```diff
@@ -0,0 +1,71 @@
+use std::env;
+use std::process::Command;
+
+use failure::{Error, ResultExt};
+use clap::{Arg, App, SubCommand, AppSettings};
+
+use lock::Server;
+use super::exit_with;
+
+pub fn run() -> Result<(), Error> {
+    let matches = App::new("Cargo Fix")
+        .bin_name("cargo")
+        .subcommand(
+            SubCommand::with_name("fix")
+                .version(env!("CARGO_PKG_VERSION"))
+                .author("The Rust Project Developers")
+                .about("Automatically apply rustc's suggestions about fixing code")
+                .setting(AppSettings::TrailingVarArg)
+                .arg(Arg::with_name("args").multiple(true))
+                .arg(
+                    Arg::with_name("broken-code")
+                        .long("broken-code")
+                        .help("Fix code even if it already has compiler errors")
+                )
+        )
+        .get_matches();
```

Comment:
Looks good. I tend use structopt myself as it makes matching on subcommands and parsing args to types so much easier, but it's not a big deal here